1_OCaml/TP/TP1.ipynb

```diff
@@ -20,6 +20,42 @@
     "Puis utilisez ces fonctions sur des arguments de votre choix."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        let c = float(b)**k
</pre><p>let c = float(b)**k in </p><p>c;;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>